.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
+<!--- If you're describing a bug, tell us what should happen -->
+<!--- If you're suggesting a change/improvement, tell us how it should work -->
+
+## Current Behavior
+<!--- If describing a bug, tell us what happens instead of the expected behavior -->
+<!--- If suggesting a change/improvement, explain the difference from current behavior -->
```

Comment:
The lines 5 and 9 are partly the same and might cause confusion on how to use them. Erase line 9 and keep all suggestions for #Expected Behavior (instead of current behavior)